client/src/Core/Components/NavigationControlHeader.tsx

```diff
@@ -0,0 +1,67 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License.
+ *--------------------------------------------------------------------------------------------*/
+
+import { FontSizes, FontWeights, mergeStyles, styled } from '@fluentui/react';
+import { useObserver } from 'mobx-react-lite';
+import React from 'react';
+import { PublishControlArea } from '../../Features/PublishAssignment/PublishControlArea';
+import { PublishSuccessMessageBar } from '../../Features/PublishAssignment/PublishSuccessMessageBar';
+import { useStore } from '../../Stores/Core';
+import { themedClassNames } from '../Utils/FluentUI';
+import { IStylesOnly, IThemeOnlyProps, SimpleComponentStyles } from '../Utils/FluentUI/typings.fluent-ui';
+import { stickyHeaderStyle } from './Common/StickyHeaderStyle';
+import * as NavBarBase from './Navbar';
+
+type NavigationControlHeaderStyles = SimpleComponentStyles<'assignmentTitle' | 'navAndControlArea'>;
+
+function NavigationControlHeaderInner({ styles }: IStylesOnly<NavigationControlHeaderStyles>): JSX.Element {
+  const assignmentStore = useStore('assignmentStore');
+  const classes = themedClassNames(styles);
+  return useObserver(() => (
+    <>
+      <span className={classes.assignmentTitle}>{assignmentStore.assignment?.name}</span>
+      <div className={classes.navAndControlArea}>
+        <NavBarBase.NavbarTop />
+        <PublishControlArea />
+      </div>
+      <PublishSuccessMessageBar
```

Comment:
Just thinking out loud here, if we put this inside the div above, will it make the experience better?

Reply:
Moved this to future iterations based on the offline discussion.